src/components/Tooltip/TooltipTypes.d.ts

```diff
@@ -29,6 +29,7 @@ export interface ITooltip {
   positionStrategy?: PositionStrategy
   delayShow?: number
   delayHide?: number
+  styles?: object
```

Comment:
```suggestion
  styles?: CSSProperties
```

don't forget to import `CSSProperties` from `react`

Reply:
the html prop is usually named `style`. i'd suggest it over `styles`, but it's your call.

Reply:
I was thinking about plural or not for this word, but you're right, I'll update, thanks!

Reply:
@gabrieljablonski updated